Cargo.toml

```diff
@@ -71,8 +71,7 @@ env_logger = "0.8.2"
 rand = "0.8.5"
 
 [features]
-fusedev = ["fuse-backend-rs/fusedev"]
-virtiofs = ["fuse-backend-rs/vhost-user-fs", "vm-memory", "vhost", "vhost-user-backend", "virtio-queue", "virtio-bindings"]
+default = ["fuse-backend-rs/fusedev", "fuse-backend-rs/vhost-user-fs", "vm-memory", "vhost", "vhost-user-backend", "virtio-queue", "virtio-bindings"]
```

Comment:
We can't change the default feature this way. Due to limitation of cargo.toml, it will break macos builds. We could achieve the same by changing Makefile instead.
'''
default=["fusedev"]
fusedev = ["fuse-backend-rs/fusedev"]
virtiofs = ["fuse-backend-rs/vhost-user-fs", "vm-memory", "vhost", "vhost-user-backend", "virtio-queue", "virtio-bindings"]
'''